lardataobj/Simulation/ParticleAncestryMap.cxx

```diff
@@ -0,0 +1,41 @@
+#ifndef PARTICLEANCESTRYMAP_CXX
+#define PARTICLEANCESTRYMAP_CXX
+
+#include "lardataobj/Simulation/ParticleAncestryMap.h"
+
+namespace sim {
+
+  void ParticleAncestryMap::SetMap(const std::map<int, std::set<int>>& map) { fParticleMap = map; }
+
+  std::map<int, std::set<int>> ParticleAncestryMap::GetMap() const { return fParticleMap; }
```

Comment:
Recommend returning by `const` reference to avoid unnecessary copies.

```suggestion
  std::map<int, std::set<int>> const& ParticleAncestryMap::GetMap() const { return fParticleMap; }
```